src/app/util/attribute-storage.cpp

```diff
@@ -773,9 +773,9 @@ static uint8_t findClusterEndpointIndex(EndpointId endpoint, EmberAfClusterId cl
         {
             break;
         }
-        epi = static_cast<uint8_t>(epi +
-                                           (emberAfFindClusterIncludingDisabledEndpointsWithMfgCode(
-                                                emAfEndpoints[i].endpoint, clusterId, mask, manufacturerCode) != NULL)
+        epi = static_cast<uint8_t>((epi +
```

Comment:
This does not look right; it looks like this got broken in #3681; thank you for catching it.  The right thing should be:
```
        epi = static_cast<uint8_t>(epi +
                                           ((emberAfFindClusterIncludingDisabledEndpointsWithMfgCode(
                                                emAfEndpoints[i].endpoint, clusterId, mask, manufacturerCode) != NULL)
                                       ? 1
                                       : 0));
```

That is, add either 1 or 0 to `epi` depending on the complicated condition.